collect_app/src/test/java/org/odk/collect/android/openrosa/OpenRosaResponseParserImplTest.kt

```diff
@@ -0,0 +1,39 @@
+package org.odk.collect.android.openrosa
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Test
+import org.kxml2.io.KXmlParser
+import org.kxml2.kdom.Document
+import org.xmlpull.v1.XmlPullParser
+import java.io.StringReader
+import java.lang.StringBuilder
+
+class OpenRosaResponseParserImplTest {
```

Comment:
I think we should have more test here (it might be a separate issue). It looks like we haven't had `OpenRosaResponseParserImpl` covered with tests and you just added one for your case.

Reply:
Yeah I think we pulled it out for testing the integration between it and `OpenRosaFormSource` so it never got its own tests (this was all originally untested code). This would be a nice object to backfill though, so I'll create an issue as you suggest.

Reply:
https://github.com/getodk/collect/issues/4919